packages/site-demo/content/product/components/message/react/closable.tsx

```diff
@@ -0,0 +1,11 @@
+import { Message, Kind } from '@lumx/react';
+import React from 'react';
+
+export const App = () => (
+    <Message kind={Kind.info} hasBackground closeButtonProps={{ label: 'Close', onClose: () => console.log('Close') }}>
+        <p>
+            Message text quisque tincidunt lobortis dui non auctor. Donec porta, ligula volutpat vehicula aliquet, dui
+            sapien tempus felis, sed cursus diam ante.
+        </p>
+    </Message>
+);
```

Comment:
I'm not sure we should demo like this. I think we should make it interactive and have it actually close 
But then maybe we need a button to reopen the message when it is closed 🤔

Reply:
That seems to be a good idea, I didn't want to suggest that the component handles the "close" part. But now that you mention it, I realize that we already demo some components this way so, I guess it's ok